nixos/modules/services/web-apps/dolibarr.nix

```
@@ -50,6 +50,16 @@ in
   options.services.dolibarr = {
     enable = mkEnableOption (lib.mdDoc "dolibarr");
 
+    package = mkOption {
+      type = types.package;
+      default = pkgs.dolibarr;
+      defaultText = literalExpression "pkgs.dolibarr";
+      example = literalExpression "unstable.dolibarr";
+      description = lib.mdDoc ''
+        Dolibarr package to use.
+      '';
+    };
+
```

We have a neat shorthand for this that's a bit underused at the moment, but we're trying to migrate towards using it almost everywhere.

```suggestion
    package = mkPackageOptionMD pkgs "dolibarr" { };
```

---

Thanks, I switched to it. Much simpler now :) I remember having seen it in the manual but I wasn't sure if it's endorsed.